weed/s3api/s3err/s3api_errors.go

```diff
@@ -157,6 +158,11 @@ var errorCodeResponse = map[ErrorCode]APIError{
 		Description:    "Argument max-parts must be an integer between 0 and 2147483647",
 		HTTPStatusCode: http.StatusBadRequest,
 	},
+	ErrInvalidMaxDeleteObjects: {
+		Code:           "InvalidArgument",
+		Description:    "Argument objects must be contains a list of up to 1000 keys",
```

Comment:
how about this description?

Argument objects can contain a list of up to 1000 keys

Reply:
@chrislusf  done